internal/pkg/config/types.go

```diff
@@ -19,6 +19,19 @@ import (
 	"github.com/edgexfoundry/go-mod-core-contracts/clients"
 )
 
+func ListDefaultServices() map[string]string {
```

Comment:
It's related because the SMA does the same thing. This change prevents duplicate implementation for listing all relevant services. If the list of services needs to be managed it can be done in one spot.

Reply:
Okay, I see it is related now, but I still think that in the future we should try to split off these kinds of small changes into separate PR's because they are more easily reviewed and the smaller we can make ~3000 line PRs the better 

Fine to leave as is, my comment about splitting into it's own commit still applies though